lib/matrix_api_lite/generated/api.dart

```diff
@@ -4591,13 +5056,18 @@ class Api {
   /// which has not yet landed in the spec. For example, a feature currently
   /// undergoing the proposal process may appear here and eventually be taken
   /// off this list once the feature lands in the spec and the server deems it
-  /// reasonable to do so. Servers may wish to keep advertising features here
-  /// after they've been released into the spec to give clients a chance to
-  /// upgrade appropriately. Additionally, clients should avoid using unstable
-  /// features in their stable releases.
+  /// reasonable to do so. Servers can choose to enable some features only for
+  /// some users, so clients should include authentication in the request to
+  /// get all the features available for the logged-in user. If no
+  /// authentication is provided, the server should only return the features
+  /// available to all users. Servers may wish to keep advertising features
+  /// here after they've been released into the spec to give clients a chance
+  /// to upgrade appropriately. Additionally, clients should avoid using
+  /// unstable features in their stable releases.
   Future<GetVersionsResponse> getVersions() async {
     final requestUri = Uri(path: '_matrix/client/versions');
     final request = Request('GET', baseUri!.resolveUri(requestUri));
+    request.headers['authorization'] = 'Bearer ${bearerToken!}';
```

Comment:
This looks wrong

Reply:
Yes, but that should be optional. We call this before login!

Reply:
oh right!